cli/src/cli.rs

```diff
@@ -42,6 +42,14 @@ pub enum CreateCommands {
     Agenda,
 }
 
+#[derive(Debug, Subcommand)]
+pub enum PeerCommand {
+    Add { address: String },
+    Remove { name: String },
+    Update,
```

Comment:
What is this command for?

Reply:
Synchronous fetch/broadcast of peer discovery protocol. This may add some remote repositories

Reply:
ok